src/tests/ftest/server/daos_server_dump.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from __future__ import print_function
+
+from apricot import TestWithServers
+from server_utils import ServerFailed
```

Comment:
(pylint-unused-import) Unused ServerFailed imported from server_utils

Reply:
Should this be fixed?